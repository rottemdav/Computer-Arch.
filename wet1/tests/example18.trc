16 6 1 0 global_history global_tables using_share_lsb
0x9841364 T 0x9a46de4
0xa14c5e90 N 0xaf0
0x8f8 N 0x94870ef8
0x7b61818 T 0x653d7a8
0xe791b8 T 0x4858
0xbc66b534 T 0xbea64bc
0xb945a44 T 0xe4a8c
0x81ee70 N 0x5697cf98
0x8f8 N 0x94870ef8
0x6a7c N 0x8424
0x150c T 0x4a6e0
0xe791b8 N 0x4858
0x7a0 N 0x2546e8
0xa14c5e90 N 0xaf0
0xa14c5e90 T 0xaf0
0xc4fbc N 0x36e8
0x5c99ac0 N 0xa14
0x989bc60 N 0xa134
0x989bc60 T 0xa134
0x989bc60 N 0xa134
0x6a7c T 0x8424
0xd2c N 0x504c0
0x9841364 T 0x9a46de4
0xa14c5e90 N 0xaf0
0xe791b8 N 0x4858
0x6a7c T 0x8424
0x7a0 N 0x2546e8
0x39c N 0x628
0xc4fbc N 0x36e8
0x150c T 0x4a6e0
0xb945a44 N 0xe4a8c
0xbc66b534 N 0xbea64bc
0xe74ccc N 0x138c353c
0xbc66b534 T 0xbea64bc
0x7e4d4 T 0x7f0ce0
0xe5640c38 T 0x25b3427c
0xe791b8 T 0x4858
0x7e4d4 N 0x7f0ce0
0xfe14b650 N 0x5d7b518
0x9841364 T 0x9a46de4
0x61bdd0 T 0xc4020
0x989bc60 T 0xa134
0x39c T 0x628
0xb945a44 T 0xe4a8c