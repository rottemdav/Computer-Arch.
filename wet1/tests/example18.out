0x9841364 N 0x9841368
0xa14c5e90 N 0xa14c5e94
0x8f8 N 0x8fc
0x7b61818 N 0x7b6181c
0xe791b8 N 0xe791bc
0xbc66b534 N 0xbc66b538
0xb945a44 N 0xb945a48
0x81ee70 N 0x81ee74
0x8f8 N 0x8fc
0x6a7c N 0x6a80
0x150c N 0x1510
0xe791b8 N 0xe791bc
0x7a0 N 0x7a4
0xa14c5e90 N 0xa14c5e94
0xa14c5e90 N 0xa14c5e94
0xc4fbc N 0xc4fc0
0x5c99ac0 N 0x5c99ac4
0x989bc60 N 0x989bc64
0x989bc60 N 0x989bc64
0x989bc60 N 0x989bc64
0x6a7c N 0x6a80
0xd2c N 0xd30
0x9841364 N 0x9841368
0xa14c5e90 N 0xa14c5e94
0xe791b8 N 0xe791bc
0x6a7c N 0x6a80
0x7a0 N 0x7a4
0x39c N 0x3a0
0xc4fbc N 0xc4fc0
0x150c N 0x1510
0xb945a44 N 0xb945a48
0xbc66b534 N 0xbc66b538
0xe74ccc N 0xe74cd0
0xbc66b534 N 0xbc66b538
0x7e4d4 N 0x7e4d8
0xe5640c38 N 0xe5640c3c
0xe791b8 N 0xe791bc
0x7e4d4 N 0x7e4d8
0xfe14b650 N 0xfe14b654
0x9841364 N 0x9841368
0x61bdd0 N 0x61bdd4
0x989bc60 N 0x989bc64
0x39c N 0x3a0
0xb945a44 N 0xb945a48
flush_num: 21, br_num: 44, size: 646b
